subsys/bluetooth/host/classic/br.c

```diff
@@ -907,6 +911,42 @@ int bt_br_init(void)
 		}
 	}
 
+	buf = bt_hci_cmd_create(BT_HCI_OP_READ_DEFAULT_LINK_POLICY_SETTINGS, 0);
+	if (!buf) {
+		return -ENOBUFS;
+	}
+
+	err = bt_hci_cmd_send_sync(BT_HCI_OP_READ_DEFAULT_LINK_POLICY_SETTINGS, buf, &rsp);
+	if (err) {
+		return err;
+	}
+
+	rp = (void *)rsp->data;
+	default_link_policy_settings = rp->default_link_policy_settings;
+
+	bool should_enable = CONFIG_BT_DEFAULT_ROLE_SWITCHABLE;
+	bool is_enabled = default_link_policy_settings &
+			  BT_HCI_LINK_POLICY_SETTINGS_ENABLE_ROLE_SWITCH;
```

Comment:
Use `()` here. Added benefit is that editors will do the split line alignment more correctly.